### DISCUSSION TITLE
ios & desktop

### DISCUSSION THREAD

**Discussion by trymeouteh** [sadellie/unitto#86] [Ideas]
Would be amazing to have this app on ios, windows, mac and linux.

Perhaps using flutter or tauri to ship the app for all popular devices?

**Comment by inson1** [comment]
I think its out of scope of the project. It would be necessary to rewrite the entire project. I think its better to add more features and "finish" the calculator than create new calculator from scratch.

But I would Iove to have some open source desktop calculator!

**Comment by sadellie** [comment]
Sounds expensive
